portality/tasks/harvester_helpers/runner.py

```diff
@@ -84,8 +84,9 @@ def run_only_once():
         flask.logging.create_logger(app)
 
     accs = list(app.config.get("HARVESTER_API_KEYS", {}).keys())
```

Comment:
An observation here - we still use `HARVESTER_API_KEYS` which is a dictionary of `{user_id: doaj_api_key}`, but we're no longer creating items via the API. the configuration structure no longer makes sense. (no problem they're just ignored) We may wish to change this to contain which harvester(s) to run per account.

We might want to consider adding a role 'harvester' so it's easier to see and manage which accounts retrieve via the harvester.